src/ripple/app/tx/impl/NFTokenMint.cpp

```diff
@@ -31,16 +31,23 @@
 
 namespace ripple {
 
+static std::uint16_t
+extractNFTokenFlagsFromTxFlags(std::uint32_t txFlags)
```

Comment:
const?

Reply:
It could be `const` with absolutely no harm.  Typically literals like this are passed as non-`const` since no harm will come to the calling function if they get modified in the called function.  Since it is a literal, not a reference, this is a copy.  In C++ it is usually cheaper compute-wise to pass small objects, like a `uint32_t`, by value.

So, I'm happy to add the `const` if you would like.  But I don't think it increases safety in this particular case.

Reply:
cool! i agree we we don't need change here 👍